### PR TITLE
SP-182: Publish Flu-Vacc model in community library

### PR DESCRIPTION
**Pull request by burrelln**
In addition to the usual community library submission requirements, please also look at the following additional criteria: 
1. credit must be given to the model’s original authors 
2. the initial [publication](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030085) and NetLogo implementation must be referenced in the model's description.

Let me know if you think either of these criteria are not sufficiently met with the current documentation.

### CODE REVIEW

File: Flu-Vacc/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# SynthPop FluVacc Model
+
+## Authors
+
+Noah Burrell (noah.burrell@epistemix.com)
+
+## Description
+
+This model is an adaptation of a model originally implemented using NetLogo. The NetLogo implementation provides the following description:
```
Comment (lanecodes) [inline Flu-Vacc/README.md:9]: Was the quoted text below from a website or a paper? If a website, I suggest linking to that site from the text `NetLogo implementation`. If a paper, a citation would be appropriate here. This is out of an abundance of caution relating to the additional criteria noted in the PR description.

Comment (burrelln) [reply Flu-Vacc/README.md:9]: The quoted text is from the NetLogo model itself (the actual `.nlogo` file), which has a description included. AFAIK, that file does not exist somewhere that I could link to it, it was shared with me via email. But I can add citations to the paper in the first sentence and clarify the second sentence with something like "alongside the code"

Comment (lanecodes) [reply Flu-Vacc/README.md]: Perfect

**Review by lanecodes — APPROVED** [review]
Nice, looks good to me!

**Pull request merged**